Makefile

```diff
@@ -114,21 +118,28 @@ e2e.test:
 test-e2e: e2e.test
 	./e2e.test -test.timeout 180m -test.v -ginkgo.v
 
-test-go: generate
-	go build -tags containers_image_openpgp ./...
+test-go: generate build-all validate-go lint-go unit-test-go coverage-go
 
+.PHONY: validate-go
+validate-go:
 	gofmt -s -w cmd hack pkg test
 	go run ./vendor/golang.org/x/tools/cmd/goimports -w -local=github.com/Azure/ARO-RP cmd hack pkg test
 	go run ./hack/validate-imports cmd hack pkg test
 	go run ./hack/licenses
 	@[ -z "$$(ls pkg/util/*.go 2>/dev/null)" ] || (echo error: go files are not allowed in pkg/util, use a subpackage; exit 1)
 	@[ -z "$$(find -name "*:*")" ] || (echo error: filenames with colons are not allowed on Windows, please rename; exit 1)
 	@sha256sum --quiet -c .sha256sum || (echo error: client library is stale, please run make client; exit 1)
-	go test -tags e2e -run ^$$ ./test/e2e/...
-
 	go vet ./...
-	set -o pipefail && go test -v ./... -coverprofile cover.out | tee uts.txt
```

Comment:
Now that we don't need `pipefail` we can remove `SHELL = /bin/bash` from the `Makefile`. See e343586d3607b086f2fcdbbe2dbd2e3be9394532